corehq/apps/hqwebapp/static/hqwebapp/less/_hq/forms.less

```diff
@@ -462,3 +462,81 @@ table .ko-inline-edit .read-only:hover {
 .radio-select_label-placeholder {
   height: @input-height-base;
 }
+
+
+.hs-input {
```

Comment:
`hubspot` would be better than `hs`

Reply:
(not blocking)

Reply:
it's the style that hubspot inserts. I can't change it unfortunately.